### PR TITLE
Search task07

### PR DESCRIPTION
**Pull request by Sina-pz**
Description:

- created search bar version 01

### CODE REVIEW

**Comment by veeranjaneyuludavuluri** [comment]
good to go.

File: src/index.html [hunk 1]
```diff
@@ -10,7 +10,12 @@
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.3/css/fontawesome.min.css" integrity="sha384-wESLQ85D6gbsF459vf1CiZ2+rr+CsxRY0RpiF1tLlQpDnAgg6rwdsUF1+Ics2bni" crossorigin="anonymous">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
```
Comment (veeranjaneyuludavuluri) [inline src/index.html:16]: I think we can overcome this with angular material which is already included in our project

Comment (Sina-pz) [reply src/index.html:16]: I added this comment as and TODO  in this branch and will address it in search version 2 task

**Review by dhaval2989 — APPROVED** [review]
Good Job

**Review by veeranjaneyuludavuluri — APPROVED** [review]

**Pull request merged**